browser/src/robocorp/browser/cli.py

```diff
@@ -2,6 +2,8 @@
 import argparse
 import sys
 
+from robocorp.tasks import cli  # noqa: F401
```

Comment:
Please add a comment here on why this is done.

Reply:
@fabioz Is this slight behaviour change (the warning issuing) now considered a new feature or just a hotfix attracting the PATCH version change?

Reply:
I'd say this is a new feature (for me, warning that the user does not have truststore is a feature, not a bugfix -- besides, users that don't want it will need to change their code to ignore the warning which is always a bit annoying).

Reply:
It is indeed, just that we had too many scars with the enterprise customers having this networking issue as a blocker, so that we want all to be aware of it in case is missing.